### PR TITLE
create book model

### PR DESCRIPTION
**Pull request by oksana-feshchenko**

### CODE REVIEW

File: books/models.py [hunk 1]
```diff
@@ -1,3 +1,19 @@
...
+
+class Book(models.Model):
+    title = models.CharField(max_length=255)
+    author = models.CharField(max_length=255)
+    cover = models.CharField(choices=[(tag, tag.value) for tag in CoverType], max_length=4)
+    inventory = models.PositiveIntegerField()
+    daily_fee = models.DecimalField(max_digits=6, decimal_places=2)
+
+    def __str__(self):
+        return f"{self.title} by {self.author}"
```
Comment (bihunva) [inline books/models.py]: add empty line

**Review by danylo-d — APPROVED** [review]
Add blank lines on the end

File: books/models.py [hunk 2]
```diff
@@ -1,3 +1,19 @@
+from enum import Enum
+
 from django.db import models
 
-# Create your models here.
+
+class CoverType(Enum):
+    HARD = "hard"
+    SOFT = "soft"
```
Comment (bihunva) [inline books/models.py]: it would be better to move everything related to the Book model to a class

**Review by bihunva — CHANGES_REQUESTED** [review]

**Review by danylo-d — APPROVED** [review]
Nice

**Review by bihunva — APPROVED** [review]

**Pull request merged**